app/views/__tests__/__snapshots__/Export.spec.js.snap

```diff
@@ -206,17 +206,6 @@ exports[`<ExportScreen /> renders correctly 1`] = `
               >
                 Share location data
               </Text>
-              <SvgXml
```

Comment:
missing icon, is this right?

Reply:
It would be nice to have typescript! this was due to a typo i believe.